sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -10747,7 +10743,7 @@
                 }
               ],
               "className": "alert alert-info fa fa-info-circle",
-              "content": "<strong> Info!</strong><br /> If you have not verified your Disability Status with StudentAid BC as PD or PPD and wish to receive disability type funding, you must first complete and submit an Appendix 8 form. Once StudentAid BC verifies your Disability Status, you must Request a Change on your student financial assistance application to receive disability type funding.<br>An applicant who identifies as a student with a disability and does not have their Disability Status verified with StudentAid BC will not receive any funding type (other or disability) until their Disability Status has been verified. If you wish to receive other type funding while StudentAid BC verifies your Disability Status, please choose option 2.",
+              "content": "<strong> Info!</strong><br /> Please note you must have your disability status verified by StudentAid BC to receive disability funding. If your disability status has already been verified, no further action is required.\n<br/><br/>\nIf your disability status has not yet been verified by StudentAid BC, you must complete an <a href=\"https://studentaidbc.ca/sites/all/files/form-library/appendix_8.pdf\">\nAppendix 8 - Disability Programs Application</a> to verify your permanent, or persistent or prolonged disability, and have your disability approved.\n<br/><br/>\nStudents with a permanent disability, or a persistent or prolonged disability may qualify for other programs. Contact the Accessibility Services Coordinator or financial aid office at your school, or visit <a href=\"https://studentaidbc.ca/\">\nwww.StudentAidBC.ca</a>",
```

Comment:
Those links should have some attributes like in: 
```html
<a  class="formio-href" 
    href="http://www.xyz.com" 
    target="_blank"
    rel="noopener noreferrer">Link</a>
```

Reply:
updated